packages/dashboard/src/app/views/myStories/filters/reducer.js

```diff
@@ -51,18 +51,31 @@ const reducer = (state, { type, payload = {} }) => {
       const filters = [...state.filters];
       filters[idx] = { ...filter, ...value };
 
+      // update filtersObject
+      const filtersObject = {};
+      for (const f of filters) {
```

Comment:
Nit: I'd prefer a more descriptive name than `f` for this, if `filter` isn't declared in the above scope, I feel like that would work.

Reply:
You could still use `filter` since it'll be scoped to the body of the `for` loop (and the outer `filter` isn't used in the loop), but could cause confusion. I say use a capital `F`, with authority!